### PR TITLE
feat: added permissions page

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15577569

**New stuff** ✨

* Added permissions page - individual authority tab is wired and working, multisig is only UI (data is copied from individual tab)
* Updated phosphor icons package because some icons were missing in old version

## TODO

- Change link in kebab menu to apply manage permission filter

## Out of scope

- Filtering
- MultiSig logic

Resolves [#1800](https://github.com/JoinColony/colonyCDapp/issues/1800)

### CODE REVIEW

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @CzarekDryl great start on this permissions page! 

A couple of picks ups:

1. Padding between sections should be 24px and are currently 16px. 

2. The font of the numbers inside the counters next to each heading should be 8px as per Figma.
image

This also includes the number is the tabbed menus
image

 
 3. Long titles especially for the extensions should truncate as per Figma.
 
image
 
 4. For the styling of the blocks that show in the empty state for each section, can you remove the gray border and reduce the height to 150px?
 
image

5. Kebab menu is sitting not center aligned on mobile
image

6. Manage permissions doesn't go full-width on mobile
image

7. I gave 4 people admin permissions as you can see here:
image

but when I filter by 'all teams' the do not show up. They only show when the specific team is selected.

image
 
 8. Two rows of results (except mobile) so 8 on desktop should show before pagination is triggered.
 
image

9. Pagination should be styled like members with the same padding and font colours.
image

Figma: https://www.figma.com/file/moME7kMfr5pdt6PYrfDtm6/Members?type=design&node-id=201-12367&mode=design&t=se0vUPpjlLuNJ4k4-4


10. Dropdown should be styled on mobile to match Figma. Width is too narrow.
image

10. Close button is missing on mobile modal
image

11. Icon in the modal is too big
image

12. Link styling inside the modal should match Figma. It should be gray in the default state and then blue on hover
image

File: src/components/v5/shared/CountBox/CountBox.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { type FC } from 'react';
+
+import { type CountBoxProps } from './types.ts';
+
+const CountBox: FC<CountBoxProps> = ({ count }) => (
+  <span className="flex items-center justify-center text-blue-400 text-[0.5rem] font-bold bg-blue-100 rounded-sm ml-1.5 w-[0.75rem] h-[0.75rem] pointer-events-none">
```
Comment (ArmandoGraterol) [inline src/components/v5/shared/CountBox/CountBox.tsx:6]: `w-[0.75rem]` and `h-[0.75rem]` can be replaced by `w-3` and `h-3`

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Code-wise we seem to be good, I didn't notice anything too out of the ordinary, just some classes that can replaced by the ones that tailwind already provide

File: src/components/v5/shared/CountBox/CountBox.tsx (hunk 1 above)
Comment (CzarekDryl) [reply src/components/v5/shared/CountBox/CountBox.tsx]: I had to change it to min-width

File: src/components/v5/shared/CountBox/CountBox.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, { type FC } from 'react';
+
+import { type CountBoxProps } from './types.ts';
+
+const CountBox: FC<CountBoxProps> = ({ count }) => (
+  <span className="flex items-center justify-center text-blue-400 text-[0.5rem] font-bold bg-blue-100 rounded-sm ml-1.5 min-w-[0.75rem] min-h-[0.75rem] px-[0.1875rem] pointer-events-none">
```
[the remark this answers is not in the document]
Comment (CzarekDryl) [reply src/components/v5/shared/CountBox/CountBox.tsx:6]: I've tried it but it's not working, maybe we have old tailwind version or something

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @CzarekDryl thanks for working this all my requested updates.

Just one further adjustment: 

1. The mobile menu triggered from the kebab should match the content container width. Currently it exceeds the width of the content.

**Review by arrenv — CHANGES_REQUESTED** [review]
Nice work @CzarekDryl, looking good.

A couple of things before this will be able to be merged:

1. We need to hide the "Multi-sig Authority" tab, this should be in another feature branch so that it can be wired up in future.

2. The tabs should have different routes, as of the spec, so they can be linked to directly.

3. `View permission actions` in top kebab should be hidden, as it is not filtering correctly.

### Additional work:
4. Can we please options to each card kebab menu for both members and extensions with the option "Manage permissions"
    - Opens the Action panel with the “Manage permissions” action and member or extension address pre-selected.

5. Can we update the copy in the "Permission types" modal to the following:

https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-v5?type=design&node-id=7292-383014&mode=design&t=hJ7bZDuFrLX6bk0u-4
> Permission types determine executable actions. Permissions cascade down from parent teams, allowing top-level permissions to take the same action in sub-teams.
> 
> Permissions can be assigned to any wallet or contract, enabling a Colony structure without individual control. Yet, assigning the 'Owner' permission to at least one address is vital. This role can be delegated to contracts, such as the 'Reputation’ Decision Method extension, enabling decentralized governance through Reputation.

And under Mod:
> Only available with individual permissions. Actions that can be performed:

image

image

**Review by melyndav — DISMISSED** [review]
Hey @CzarekDryl thanks for fixing the mobile menu issue. I'm approving this PR but please resolve Arren's issues before merging. Thanks 👍

**Comment by CzarekDryl** [comment]
`Can we please options to each card kebab menu for both members and extensions with the option "Manage permissions"

Opens the Action panel with the “Manage permissions” action and member or extension address pre-selected.` is going to be done in other ticket.

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me 👍

**Review by arrenv — CHANGES_REQUESTED** [review]
Nice one @CzarekDryl, thank you for the updates.

All looks good, I just ran into an error when expanding the copy in the modal:
image

**Review by arrenv — APPROVED** [review]

**Review by melyndav — APPROVED** [review]
Approving this @CzarekDryl 👍 Thanks for your work!

**Pull request merged**